lib/bulk_data_test_kit/v1.0.1/bulk_data_output_check_test.rb

```diff
@@ -24,6 +24,16 @@ class BulkDataOutputCheckTest < Inferno::Test
       DESCRIPTION
       # link 'http://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#response---complete-status'
 
+      verifies_requirements 'hl7.fhir.uv.bulkdata_2.0.0@33',
+                            'hl7.fhir.uv.bulkdata_2.0.0@34',
+                            'hl7.fhir.uv.bulkdata_2.0.0@35',
+                            'hl7.fhir.uv.bulkdata_2.0.0@36',
+                            'hl7.fhir.uv.bulkdata_2.0.0@164',
+                            'hl7.fhir.uv.bulkdata_2.0.0@167',
+                            'hl7.fhir.uv.bulkdata_2.0.0@168',
+                            'hl7.fhir.uv.bulkdata_2.0.0@170',
```

Comment:
I read 166 as definitional but have added the reference to it since it is checking for valid JSON and I think it makes more sense to include it if 165 is included and I agree that it should be.